### PR TITLE
Fixing multi-arch testcase OCP-22283

### PR DESCRIPTION
**Pull request by varad-ahirwadkar**
Updating the multi-arch testcase OCP-22283 with importMode PreserveOriginal.

### CODE REVIEW

File: features/cli/pod.feature [hunk 1]
```diff
@@ -243,7 +243,7 @@ Feature: pods related scenarios
   @critical
   Scenario: OCP-22283:Authentication 4.0 Oauth provider info should be consumed in a pod
     Given I have a project
-    When I create a new application with:
+    When I create a new application with importmode if supported with:
```
Comment (pruan-rht) [inline features/cli/pod.feature:246]: @varad-ahirwadkar where is this method defined?

Comment (varad-ahirwadkar) [reply features/cli/pod.feature:246]: Sorry, I did not see it. It is not available under this repository.

Comment (varad-ahirwadkar) [reply features/cli/pod.feature:246]: I see another PR for the same test case https://github.com/openshift/verification-tests/pull/3462.

Comment (pruan-rht) [reply features/cli/pod.feature:246]: @varad-ahirwadkar, currently, there's no such step_defintion, so you new addition will definitely fail.  Have you tested it with an actual run?  BTW, that PR that you reference is calling a different step than what you are calling.
```
When I create a new application with:
```

Comment (varad-ahirwadkar) [reply features/cli/pod.feature:246]: Yes it fails. Sorry for inconvenience. By mistake I used this method which is not part of this repository.

Comment (liangxia) [reply features/cli/pod.feature:246]: Given PR https://github.com/openshift/verification-tests/pull/3468 is merged, I think the change is no longer needed.

Comment (varad-ahirwadkar) [reply features/cli/pod.feature:246]: Thanks. I am closing this PR.

**Comment by varad-ahirwadkar** [comment]
Closing this PR since https://github.com/openshift/verification-tests/pull/3468 resolves the issue.

**Pull request closed without merging**